iroha_client_cli/src/main.rs

```diff
@@ -280,7 +280,7 @@ mod account {
     /// Register account
     #[derive(StructOpt, Debug)]
     pub struct Register {
-        /// Id of account in form `name@domain_name'
+        /// Id of account in form `name@domain_name`
```

Comment:
This is actually intentional. This docs are used for cli help print

Reply:
Only in `main.rs`?